core/lib/multivm/src/versions/vm_fast/vm.rs

```diff
@@ -460,7 +457,6 @@ impl<S: ReadStorage> Vm<S> {
         inner.state.current_frame.sp = 0;
 
         // The bootloader writes results to high addresses in its heap, so it makes sense to preallocate it.
-        inner.state.heaps[vm2::FIRST_HEAP].reserve(get_used_bootloader_memory_bytes(VM_VERSION));
```

Comment:
reservation is removed but the comment is left, maybe should also be removed. besides, what is the reason for this removal?

Reply:
It definitely shouldn't be removed here. It is removed in my PR because its performance impact is minimal and it becomes tricky to do.

Reply:
Good catch! That's essentially a merge artifact at this point.